src/node/index.ts

```diff
@@ -131,7 +131,7 @@ module.exports = (options: BlogPluginOptions, ctx: VuePressContext) => {
               /**
                * Register pagination
                */
-              const indexPath = `/${scope}/${key}/`
+              const indexPath = path ? `${path}${key}` : `/${scope}/${key}/`;
```

Comment:
`key` is duplicated.

This path is generated by `curryFrontmatterHandler `, so you can simply pass it to `resolvePaginationConfig`.